### ISSUE TITLE
thetaSketch metrics are missing percentage

### ISSUE THREAD

**Issue by dejan** [allegro/turnilo#771]
Hi, first of all thanks for all your work on amazing tool Turnilo!

I've noticed thetaSketch metrics are missing `% Percent` selection in Turnilo.

Is there some limitation for this, or is it just a bug? Is there a workaround perhaphs that can be done in the configuration for example?

**Comment by adrianmroz-allegro** [comment]
Hi!

It is intentional. Theta sketches are approximations and any numerical operation on approximated value would increase error margin. Because percents are calculated in post-transformation it would give very inaccurate results. 

Same story with "calculate".

**Comment by dejan** [comment]
> Theta sketches are approximations and any numerical operation on approximated value would increase error margin.

Does this mean splits would not "add up" to 100% for example, or that percentages would not follow estimates ratio? Or both?

**Comment by dejan** [comment]
I've tested this by adding the percentage component in the UI. 

The problem that splits don't add up to 100% of total is present, but that's also true for absolute numbers. The ratio of error with the absolute number seems to be about the same - so I'm not seeing increased error margin (or it can happen only in certian situation?).

Here's an example I've used:

<img width="652" alt="Screenshot 2021-06-14 at 09 38 25" src="https://user-images.githubusercontent.com/4494/121856323-abb84980-ccf4-11eb-92f5-eb9bb3281b2a.png">

Both colums add up to 0.9974499570660391 of the total. Both **should** add up to 1.
